avaframe/tests/test_com1DFA.py

```diff
@@ -64,6 +64,67 @@ def test_prepareRelase(tmp_path):
     assert badName == True
 
 
+def test_checkParticlesInRelease():
+    """ test if particles are within release polygon and removed if not """
+
+    # setup required input
+    releaseLine = {'Name': ['testRel'], 'Start': np.asarray([0.]), 'Length': np.asarray([5]),
+                   'x': np.asarray([0, 10., 10.989, 0., 0.]), 'y': np.asarray([0., 0., 10.989, 10., 0.0])}
+    demHeader = {}
+    demHeader['xllcenter'] = 0.0
+    demHeader['yllcenter'] = 0.0
+    demHeader['cellsize'] = 1.0
+    demHeader['noDataValue'] = -9999
+    demHeader['nrows'] = 5
+    demHeader['ncols'] = 5
+    releaseLine['header'] = demHeader
+    particles = {'x': np.asarray([2.4, 9.7, 11.4, 13.8]), 'y': np.asarray([2.4, 9.7, 11.4, 13.8]),
+                 'Npart': 4, 'm': np.asarray([1.4, 1.7, 1.4, 1.8])}
+    radius = np.sqrt(2)
+
+    # call function to be tested
+    particles = com1DFA.checkParticlesInRelease(particles, releaseLine, radius)
+    # call function to be tested
+    particles2 = {'x': np.asarray([2.4, 9.7, 11.4, 13.8, 0.0]), 'y': np.asarray([2.4, 9.7, 11.4, 13.8, 0.0]),
+                 'Npart': 5, 'm': np.asarray([1.4, 1.7, 1.4, 1.8, 1.1])}
+    particles2 = com1DFA.checkParticlesInRelease(particles2, releaseLine, 0.1)
```

Comment:
How did you choose the values? I think we need to choose more wisely the particle position to be sure that if radius is 0, a particle slightly inside stays in but that a particle slightly outside is removed. So I would make a square release and try that.

Reply:
i changed the points to be closer to the line and the radius to be samller